### PR TITLE
What's new for engine 3.14

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/whatsnew/doc/whats-new.md

### CODE REVIEW

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,479 +1,206 @@
-The release 3.13 is the result of 4 months of work involving over
-350 pull requests and touching all aspects of the engine. In particular,
-this is the minimal release needed to run the new [2020 Euro-Mediterranean 
-Seismic Hazard Model (ESHM20)](http://hazard.efehr.org/en/Documentation/specific-hazard-models/europe/eshm2020-overview/) 
-and the new [GEM China model](https://www.globalquakemodel.org/GEMNews/china-earthquake-loss-model-2021).
+The release 3.14 is the result of 3 months of work involving nearly
+200 pull requests. The major highlights are: complete vectorization of
+hazardlib, optimization of the UCERF model, and upgrade of Shapely to
+version 1.8.0 (the upgrade causes slightly changes in the numbers
```
Comment (raoanirudh) [inline doc/whats-new.md:4]: *slight

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,479 +1,206 @@
...
 
-Finally we removed the long deprecated XML exporters.
+We implemented a cache for the distances that speedups
+substantially the calculation (we are talking about a 2x
+speedup). Since the distance calculation algorithm has changed
+internally, the numbers are expected to be slightly different than
+in past versions.
 
-# Improvements to the event based calculator
+If you already have a license for the UCERF model or you are a GEM
```
Comment (mmpagani) [inline doc/whats-new.md:79]: I would remove this.

File: doc/whats-new.md [hunk 3]
```diff
@@ -1,479 +1,206 @@
...
 
-https://docs.openquake.org/oq-engine/advanced/sslt.html
+`job.ini` files containing a Byte Order Mark (BOM) where not read
+correctly, causing issues particularly on the Windows platform.
 
-The branch IDs in the gsim logic tree file are now ignored and
-the user can skip them altogether. The change was implemented
-to fix a subtle bug causing incorrect branch paths to be listed
-in the output "Realizations" in case of duplicated branch IDs.
+There was a small bug with discard_trts, such that in some cases too much
```
Comment (mmpagani) [inline doc/whats-new.md:122]: too much -> too many (?)

File: doc/whats-new.md [hunk 4]
```diff
@@ -1,479 +1,206 @@
...
+your calculation to run out of memory or to become extra-slow. Use
+this feature with care.
 
-The `custom_site_id` is now exported also by the GMF exporters.
+# New risk features
 
-When using the `--hc` option the engine was using the site collection
-of the parent calculation and ignoring the site collection of the
-child calculation: this is now fixed.
+At user request, we implemented multi-tag aggregation: it is now
```
Comment (raoanirudh) [inline doc/whats-new.md:93]: Maybe we can add the links to the issues / PRs: https://github.com/gem/oq-engine/issues/7663

File: doc/whats-new.md [hunk 5]
```diff
@@ -1,479 +1,206 @@
...
+would perform four different kinds of aggregations (by "NAME_1 &
+taxonomy", by "NAME_1", by "taxonomy" and total aggregation) and the
+aggregate risk and aggregate curves exporters would export four files
+each. The export format is the same, however while before the totals
+were exported in the same file, now they are exported in a separate
+file.
 
-https://docs.openquake.org/oq-engine/advanced/rupture-sampling.html
+We also changed how the Aggregate Risk output is computed, reverting
+back to the definition of previous versions of the engine,
```
Comment (raoanirudh) [inline doc/whats-new.md:109]: https://github.com/gem/oq-engine/pull/7708

**Review by raoanirudh — APPROVED** [review]

**Pull request merged**